### DISCUSSION TITLE
Prevent Pjax on certain form

### DISCUSSION THREAD

**Discussion by anthonyfrancq** [PaperStrike/Pjax#201] [Q&A] [answered]
Is there a way to prevent Pjax from working on a certain form where the form is inside the selector?

**Comment by PaperStrike** [comment, marked as answer]
You can try the `exclude` sub-option in [`defaultTrigger`](https://github.com/PaperStrike/Pjax/tree/v2.2.0#defaulttrigger).

**Reply by PaperStrike** [reply]
@anthonyfrancq it's worth to mention that if the submission is handled by other scripts, and that script calls [`preventDefault`](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault), then Pjax should be going to ignore the submission. If not, it's a bug and please report it as an issue.
